### ISSUE TITLE
Suggestion - Get icon and shortcut background to show up in search results

### ISSUE THREAD

**Issue by skynet01** [lukdiekm/alfred-shortcuts#3]
First of amazing workflow! May I make a suggestion to have the workflow pull the icon and and the color of the Shortcut? This would make it easy to scan the list and quickly select shortcuts from it. Right now if you got a lot of items its hard to find them in a long list 

Thanks again :)

**Comment by lukdiekm** [comment]
I agree, that would be a great improvement. At the moment it's not possible to find out the color and icon from the cli and I don't know where shortcuts are stored. As soon as this changes, I will definetly implement this.
